server/games/game.py

```diff
@@ -245,55 +244,49 @@ def has_ai(self) -> bool:
 
     @property
     def is_even(self) -> bool:
-        teams = self.team_count()
-        if FFA_TEAM in teams:    # someone is in ffa team, all teams need to have 1 player
-            c = 1
-            teams.pop(1)
-        else:
-            n = len(teams)
-            if n <= 1:    # 0 teams are considered even, single team not
-                return n == 0
-
-            # all teams needs to have same count as the first
-            c = list(teams.values())[0]
-
-        for _, v in teams.items():
-            if v != c:
-                return False
+        """
+        Returns True iff all teams have the same player count, taking into account that players on the FFA team are in individual teams.
+        Special cases:
+         - Returns True if there are zero teams.
+         - Returns False if there is a single team.
+        """
+        # FIXME: explicitly allowing a team to be set to None here.
+        # All players with team `None` are considered on one team.
+        # This is what code in production does and allows to launch games where
+        # players are missing the "Team" option.
+        # (Rating code will later raise an Error if players are missing that option.)
+        # Is this intentional?
```

Comment:
Some unit tests were relying on games being able to launch without having the "Team" option set for every player. Is that intentional?

Reply:
[bumping this]